### PR TITLE
Encd 6130 file detail title not showing

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/encoded/static/components/filegallery.js [hunk 1]
```diff
@@ -206,10 +206,7 @@ export const compileAnalyses = (analyses, files, dataFormat = null, allowMixed =
         // Get all the analysis objects that qualify for inclusion in the Pipeline facet.
         // More than one lab OK, as long as none of them is `UNIFORM_PIPELINE_LAB` --
         // `UNIFORM_PIPELINE_LAB` is only valid if alone.
-        const qualifyingAnalyses = analyses.filter((analysis) => (
-            analysis.assembly
-            && (allowMixed || (analysis.assembly !== 'mixed' && analysis.genome_annotation !== 'mixed'))
-        ));
+        const qualifyingAnalyses = analyses.filter((analysis) => (allowMixed || (analysis.assembly && (analysis.assembly !== 'mixed' && analysis.genome_annotation !== 'mixed'))));
```
Comment (zoldello) [inline src/encoded/static/components/filegallery.js:209]: @forresttanaka let's have a discussion about this if you are open to it.

Comment (forresttanaka) [reply src/encoded/static/components/filegallery.js:209]: I know not part of this ticket, but I don’t know when you’d use `allowMixed` and when you wouldn’t; no documentation.

**Review by forresttanaka — APPROVED** [review]
Overall I think it makes sense that you simply allow for cases without an assembly to be treated much like those cases that do.

File: src/encoded/static/components/filegallery.js (hunk 1 above)
Comment (zoldello) [reply src/encoded/static/components/filegallery.js:209]: I am making a new ticket for this. Here is a medal for pointing this out - 🏅

**Pull request merged**